### PR TITLE
new release process documentation

### PR DESCRIPTION
**Pull request by davidkline-ms**
this change updates the process the WindowsDevicePortalWrapper undergoes to create a release. in place of developing and stabilizing in the master branch, this proposes to use Development and Stabilization branches similar to how the MixedRealityToolkit-Unity repository is manage.

### CODE REVIEW

File: ReleaseProcess.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+## Release Steps
+1. Declare that a release is feature complete.
+
+This will done by / in coordination with the Windows Device Portal Wrapper maintainers.
+
+2. Create a stabilization branch (ex: Stabilization_v1.2.3) off of Development
+
+Bug fixes get submitted to the stabilization branch while work on the next release continues in Development. Note: While not expected in every release, there _may_ be some cherry-picking of individual commits from Development.
+
+3. When the release has stabilized, a pull request is issued from the stabilization branch to master and it is merged.
```
Comment (NeerajW) [inline ReleaseProcess.md]: Is there any guidance on how long the stabilization period is? For MixedRealityToolkit-Unity, we say it's 1 week.

File: ReleaseProcess.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+During this time, the release is compiled, signed, packaged and submitted to NuGet.
+
+4. The master branch is tagged with the release's version number.
+
+5. A pull request is issued from the master branch into development and it is merged.
+
+No squashing is necessary on this merge.
+
+6. The stabilization branch is, optionally, deleted.
```
Comment (NeerajW) [inline ReleaseProcess.md]: What is the goal of keeping the stabilization branch and not deleting? Do you intent to service it? If not then folks can just sync to the git TAG of the release.

File: README.md [hunk 3]
```diff
@@ -3,17 +3,21 @@ A client library that wraps the [Windows Device Portal](https://msdn.microsoft.c
...
+
+```
+Note: The contribution guidelines were updated in January 2018. Pull requests that target the master branch will be asked to be retargeted to the Development branch prior to approval.
+```
 
 ---
 ---
 
-For source code documentation, see https://microsoft.github.io/WindowsDevicePortalWrapper/annotated.html
+For source code documentation, see https://microsoft.github.io/WindowsDevicePortalWrapper/annotated.html.
```
Comment (hpsin) [inline README.md]: Period doesn't impact the link?

Comment (davidkline-ms) [reply README.md]: period is not an issue.

**Pull request merged**